packages/x-charts-pro/src/internals/plugins/useChartProExport/common.ts

```diff
@@ -0,0 +1,61 @@
+import ownerDocument from '@mui/utils/ownerDocument';
+
+export function createExportIframe(title?: string): HTMLIFrameElement {
+  const iframeEl = document.createElement('iframe');
+  iframeEl.style.position = 'absolute';
+  iframeEl.style.width = '0px';
+  iframeEl.style.height = '0px';
+  iframeEl.title = title || document.title;
+  return iframeEl;
+}
+
+export function loadStyleSheets(document: Document, element: HTMLElement | SVGElement) {
```

Comment:
This looks close to the data grid print logic, maybe something to share at one point.

Reply:
Yes, it's a direct copy. I'll look into how we can share this.

Reply:
Here it is: https://github.com/mui/mui-x/pull/17548